### PR TITLE
Material for Week 4 - first pass

### PR DESCRIPTION
**Pull request by shiffman**
I'll be working on material work week 4 here. This goes along with #38.

### CODE REVIEW

File: 04_diy_neural_network/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Consider how to frame the problem and collect data.
+    * Understand critical questions to ask (e.g. Who is this for? What’s the context?)
+    * Understand the questions to ask about sourcing and collecting data.
+    * Learn how to prepare a data set, including how to normalize and properly format it.
+* Diagram the components of a two layer "vanilla" neural network.
+
+### Related Projects
+* [Gender Shades: How well do IBM, Microsoft, and Face++ AI services guess the gender of a face?](http://gendershades.org/) by Joy Buolamwini and Timnit Gebru
+* [Feminist Data Set](https://carolinesinders.com/feminist-data-set/) by Caroline Sinders
+
```
Comment (ellennickles) [inline 04_diy_neural_network/README.md:30]: This project, live until Friday, September 27, 2019, and it’s accompanying paper (which I suggest adding below) is a timely follow-up to the students' investigations into ImageNet and COCO in the previous weeks. It's critical project, especially in conversation with Fei-Fei Li's opinion piece also posted on the syllabus for this week.
```suggestion
*[ImageNet Roulette](https://imagenet-roulette.paglen.com) by Kate Crawford and Trevor Paglen with Leif Ryge
```

File: 04_diy_neural_network/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### Reading and Viewing
+* [This is how AI bias really happens—and why it’s so hard to fix](https://www.technologyreview.com/s/612876/this-is-how-ai-bias-really-happensand-why-its-so-hard-to-fix/) by Karen Hao
+* [How to Make A.I. That’s Good for People](https://www.nytimes.com/2018/03/07/opinion/artificial-intelligence-human.html) by Fei-Fei Li
+* [Estimating the success of re-identifications in incomplete datasets using generative models](https://www.nature.com/articles/s41467-019-10933-3) from nature.com
+* Video: [Analyzing & Preventing Unconscious Bias in Machine Learning](https://www.infoq.com/presentations/unconscious-bias-machine-learning) by Rachel Thomas
+
```
Comment (ellennickles) [inline 04_diy_neural_network/README.md:35]:
```suggestion
* [Excavating AI: The Politics of Images in Machine Learning Training Sets](https://www.excavating.ai) by Kate Crawford and Trevor Paglen
```

File: 04_diy_neural_network/README.md (hunk 1 above)
Comment (ellennickles) [reply 04_diy_neural_network/README.md:30]: Oh wow! On repeated spins, ImageNet Roulette labeled me a first offender, mediatrix, whiteface, lass, psycholinguist, and face.

**Comment by shiffman** [comment]
This is by no means complete and I hope to work on it more tomorrow before class and certainly quite a bit between Monday and Wednesday. Merging for now - please add any comments, suggestions, etc as new issues or pull requests.

**Pull request merged**